pwiz_tools/Skyline/Util/DataSourceUtil.cs

```diff
@@ -130,6 +130,37 @@ public static string GetSourceType(FileInfo fileInfo)
             }
         }
 
+        /// <summary>

```

Comment:
Why not use IsDataSource(string path)? Because you are trying to figure this out for something in a ZIP file? Let's not do this. It's not really valid, because to know if a directory is a data source, you need to look inside it. Doing this correctly for a ZIP file would be more complicated, and you shouldn't need to do this if you organize the tutorial ZIP file consistent with all other tutorial ZIP files.

Reply:
Yeah, already torn out since it was only to deal with a malformed zip file.

Reply:
When I think about the ultimate UI for this, which it seems like we are not engaged in designing, I think it would be nice to be able to have the option to either pause at the moment of taking the covershot, which makes it easier to tweak, e.g. adjust the layout, File > Save, and grab the .view file, or just take screenshots in bulk. I guess I personally transitioned from producing covershots to taking them all in bulk, and kinda rushed to get back to other work once this largish undertaking was complete. So, as you are pointing out, I didn't leave it in an ideal condition for the next single covershot to be added.